examples/playback_capture.rs

```diff
@@ -15,7 +15,7 @@ fn main() {
     let in_b = client.register_port("rust_in_r", j::AudioInSpec::default()).unwrap();
     let mut out_a = client.register_port("rust_out_l", j::AudioOutSpec::default()).unwrap();
     let mut out_b = client.register_port("rust_out_r", j::AudioOutSpec::default()).unwrap();
-    let process_callback = move |ps: &j::ProcessScope| -> jack::JackControl {
+    let process_callback = move |_: &j::WeakClient, ps: &j::ProcessScope| -> jack::JackControl {
```

Comment:
Yup, it's just getting passed as an &param.

It'll have all [this](https://wmedrano.github.io/rust-jack/jack/client/trait.JackClient.html) functionality.